ntex-fileadapter-acl/Cargo.toml

```diff
@@ -7,6 +7,6 @@ workspace = ".."
 
 [dependencies]
 ntex = "0.1"
-casbin = "0.9"
+casbin = "1.1"
```

Comment:
Maybe using `casbin = { version = "1.1.0", default-features = false, features = ["incremental"] }` is better